### PR TITLE
Add utilisation for electricity storages

### PR DESCRIPTION
**Pull request by sebwehrle**
We need to exchange the utilisation of electricity storage and hence require a corresponding variable. Storage utilisation is understood as discharged energy / discharge capacity.

### CODE REVIEW

File: definitions/variable/utilisation.yaml [hunk 1]
```diff
@@ -6,4 +6,8 @@
     unit: h/a
 - Utilisation|Heat Plant|Heat|{Heat Input}:
     description: utilisation / full load hours of heat plants producing heat from {Electricity Input}
+    unit: h/a
+- Utilisation|Storage|Discharging|Electricity|{Electricity Storage Types}:
```
Comment (danielhuppmann) [inline definitions/variable/utilisation.yaml:10]: I think that inverting the order would make more sense - it's a variable for utilisation -> of a storage technology -> for electricity -> for a particular kind of activity of that technology -> for a type of technology
```suggestion
- Utilisation|Storage|Electricity|Discharging|{Electricity Storage Types}:
```

Comment (danielhuppmann) [reply definitions/variable/utilisation.yaml:10]: Sounds good to me - I guess this hasn't been used anywhere yet...?

Comment (sebwehrle) [reply definitions/variable/utilisation.yaml]: I implemented and committed the suggested changes

**Review by danielhuppmann — APPROVED** [review]
Thank you, very nice!

**Pull request merged**